sdk/tests/program_macros.rs

```diff
@@ -0,0 +1,207 @@
+use bincode::serialize;
+use serde_derive::{Deserialize, Serialize};
+use solana_sdk::{
+    instruction::{AccountMeta, Instruction},
+    pubkey::Pubkey,
+};
+use solana_sdk_program_macros::instructions;
+
+#[instructions(test_program::id())]
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub enum TestInstruction {
+    /// Transfer lamports
+    #[accounts(
+        from_account(SIGNER, WRITABLE, desc = "Funding account"),
+        to_account(WRITABLE, desc = "Recipient account")
+    )]
+    Transfer {
+        /// Test a field comment
+        lamports: u64,
+    },
+
+    /// Provide one required signature and a variable list of other signatures
+    #[accounts(
+        required_account(WRITABLE, desc = "Required account"),
+        signers(SIGNER, multiple, desc = "Signer")
```

Comment:
Can this be `multiple = true`?  `multiple` on its own (within a macro) looks like the creation of an identifier.

Reply:
Same for `optional`?

Reply:
Yes